media-libs/vulkan-loader/vulkan-loader-1.3.268.ebuild

```diff
@@ -57,4 +65,16 @@ multilib_src_install() {
 	keepdir /etc/vulkan/icd.d
 
 	cmake_src_install
+
+	#Set default Vulkan Loader
+	#Intel
+	use video_cards_intel && newenvd "${FILESDIR}"/99-vulkan-icd-loader-intel 99-vulkan-icd-loader
+	#AMD
+	(use video_cards_amdgpu || use video_cards_radeonsi) && newenvd "${FILESDIR}"/99-vulkan-icd-loader-amd 99-vulkan-icd-loader
+	#Nvidia
```

Comment:
i'm very skeptical that this is correct and handles all cases. i'm not really sure why we need to do anything. why can't users of dual-gpu systems just set the envvar to pick a card to use? don't they already have to do that with DRI_PRIME?

Reply:
The problems with DRI_PRIME is also check for nouveau driver if nouveau is not install, some application refuse to launch  (Lutris do that)

Setting VK_ICD_FILENAMES dont have that problem

This is only problem if someone have the AMDGPU-PRO install (so the files created for intel probably are not needed), for some reason when  VK_ICD_FILENAMES is not set it pick the frist icd loader for alphabetical order one picking the close source one.

Reply:
We can try another approach:
- Revert the change made media-libs/vulkan-loader and add a warning to the user to set VK_ICD_FILENAMES when installing media-libs/amdgpu-pro-vulkan

Reply:
Reading the wiki also found out that setting this AMD_VULKAN_ICD=RADV or DISABLE_LAYER_AMD_SWITCHABLE_GRAPHICS_1=1 everything work as intended.

Even launching ffmpeg or obs to use amf work when one of the two is set

AMD_VULKAN_ICD have two values RADV and AMDVLK probably is better to use DISABLE_LAYER_AMD_SWITCHABLE_GRAPHICS_1=1 instead

I gonna change the PR to do that